docs/reference/release-notes/highlights.asciidoc

```diff
@@ -15,6 +15,27 @@ For detailed information about this release, see the <<es-release-notes>> and
 // should be featured in the Stack Installation and Upgrade Guide:
 
 // tag::notable-highlights[] 
+[discrete]
+=== Security features are enabled and configured by default
+
+Running {es} without security leaves your cluster exposed to anyone who can send
+network traffic to {es}. In previous versions, you had to explicitly enable the 
+{es} security features such as authentication, authorization, and network
+encryption (TLS). Starting in {es} 8.0, security is
+{ref}/configuring-stack-security.html[enabled and configured by default] when
+you start {es} for the first time. 
+
+At startup, we generate enrollment tokens so that you can connect a {kib} 
+instance or enroll additional nodes in your secured {es} cluster, without having 
+to generate security certificates or update `yml` configuration files. Just use 
+the generated enrollment token when starting new nodes or {kib} instances, and 
+the {stack} handles all of the security configuration for you. Out of the box, 
+you'll get authentication, authorization, and network encryption with TLS for 
+internode communications between {es} and {kib}. You can create new enrollment 
```

Comment:
```suggestion
internode communications and communications between {es} and {kib}. You can create new enrollment 
```

not sure if there is a more concise way to say the above but "internode" == between es nodes , which is not the same thing as between es and kibana

Reply:
I decided to change this sentence to a bulleted list and provide more detail. I think you're right that we need to call out that you get encrypted communications both between nodes and between Elasticsearch and Kibana, so the bulleted list seemed like a better option. Plus, it's easier to scan!